jenkins/automation/kubernetes/nightly-build-and-deploy.groovy

```diff
@@ -180,6 +204,7 @@ pipeline {
                     currentBuild.result = "UNSTABLE"
                 }
                 env.build_setupcortx_url = sh( script: "echo ${env.cortxcluster_build_url}/artifact/artifacts/cortx-cluster-status.txt", returnStdout: true)
+                env.changeset_log_url = sh( script: "echo ${env.changeset_log_url}artifact/CHANGESET.txt", returnStdout: true)
```

Comment:
@nikhilpatil2995, can you please archive Change set in job as well?

Reply:
It's better to archive and use that URL. As changeset job build may get removed due to log rotation.

Reply:
created